.github/workflows/deploy.yml

```diff
@@ -135,106 +160,82 @@ jobs:
-
-          aws cloudfront wait invalidation-completed --distribution-id ${{ secrets.AWS_CF_DISTRIBUTION_ID }} --id "$invalidation_id"
+          # Copy all the files to S3
+          aws s3 cp ${{ env.INTEGRATIONS_ARTIFACTS_BASE_PATH }}/legacy/js-integrations/ $s3_path_prefix/legacy/js-integrations/ $copy_args
+          aws s3 cp ${{ env.PLUGINS_ARTIFACTS_BASE_PATH }}/modern/plugins/ $s3_path_prefix/modern/plugins/ $copy_args
+          aws s3 cp ${{ env.INTEGRATIONS_ARTIFACTS_BASE_PATH }}/modern/js-integrations/ $s3_path_prefix/modern/js-integrations/ $copy_args
 
-      - name: Copy SDK artifacts to S3 (versioned directory)
+      - name: Copy SDK core artifacts to S3 (versioned directory)
         if: ${{ inputs.environment == 'production' }}
         run: |
-          core_sdk_path_prefix="packages/analytics-js/dist/cdn"
-          integration_sdks_path_prefix="packages/analytics-js-integrations/dist/cdn"
-          plugins_path_prefix="packages/analytics-js-plugins/dist/cdn"
           s3_relative_path_prefix="${{ env.CURRENT_VERSION_VALUE }}"
           s3_path_prefix="s3://${{ secrets.AWS_S3_BUCKET_NAME }}/$s3_relative_path_prefix"
-          copy_recursive_args="--recursive --cache-control ${{ env.CACHE_CONTROL_MAX_AGE }}"
-          copy_args="--cache-control ${{ env.CACHE_CONTROL_MAX_AGE }}"
-
-          integration_sdks_zip_file="all_integration_sdks.tar.gz"
-          plugins_zip_file="all_plugins.tar.gz"
-
-          integration_sdks_html_file="list.html"
-          plugins_html_file="list.html"
+          copy_args="--recursive --cache-control ${{ env.CACHE_CONTROL_MAX_AGE }}"
 
           # Copy all the files to S3
-          aws s3 cp $core_sdk_path_prefix/legacy/iife/ $s3_path_prefix/legacy/ $copy_recursive_args
-          aws s3 cp $core_sdk_path_prefix/modern/iife/ $s3_path_prefix/modern/ $copy_recursive_args
+          aws s3 cp ${{ env.CORE_ARTIFACTS_BASE_PATH }}/legacy/iife/ $s3_path_prefix/legacy/ $copy_args
+          aws s3 cp ${{ env.CORE_ARTIFACTS_BASE_PATH }}/modern/iife/ $s3_path_prefix/modern/ $copy_args
 
-          aws s3 cp $integration_sdks_path_prefix/legacy/js-integrations/ $s3_path_prefix/legacy/js-integrations/ $copy_recursive_args
-          aws s3 cp $plugins_path_prefix/modern/plugins/ $s3_path_prefix/modern/plugins/ $copy_recursive_args
-          aws s3 cp $integration_sdks_path_prefix/modern/js-integrations/ $s3_path_prefix/modern/js-integrations/ $copy_recursive_args
+      - name: Invalidate CloudFront cache for all the SDK artifacts (versioned directory)
+        if: ${{ inputs.environment == 'production' }}
+        run: |
+          invalidation_id=$(AWS_MAX_ATTEMPTS=10 aws cloudfront create-invalidation --distribution-id ${{ secrets.AWS_CF_DISTRIBUTION_ID }} --paths "/${{ env.CURRENT_VERSION_VALUE }}/*" --query "Invalidation.Id" --output text)
```

Comment:
We don't need to invalidate these since no-one knows about them yet.

Even if we discover a defect and we should release the fix under a new version.

In a rollback scenario we will need to invalidate the rolled back version paths.

Reply:
Done.